### PR TITLE
Added complex meta tests for C++ and python

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Fixed serialization and deserialization of empty vectors

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/core/src/any.cpp [hunk 1]
```diff
@@ -63,7 +63,8 @@ void Any::Base::read_to(Base& other) const {
     if (other.is<std::string>()) {
         *static_cast<std::string*>(other.addressof()) = strm.str();
     } else {
-        other.read(strm);
+        if (!strm.str().empty())
+            other.read(strm);
```
Comment (ilya-lavrenov) [inline src/core/src/any.cpp:67]: can we create dedicated ov::Any test for it? what happens if stream is empty?

File: src/bindings/python/src/pyopenvino/graph/model.cpp [hunk 2]
```diff
@@ -752,11 +752,17 @@ void regclass_graph_Model(py::module m) {
...
+            // FIXME: understand why has_rt_info causes Python crash
+            try {
+                std::vector<std::string> cpp_args(path.size());
+                for (size_t i = 0; i < path.size(); i++) {
+                    cpp_args[i] = path[i].cast<std::string>();
+                }
+                self.get_rt_info<ov::Any>(cpp_args);
+                return true;
+            } catch (ov::Exception&) {
+                return false;
```
Comment (ilya-lavrenov) [inline src/bindings/python/src/pyopenvino/graph/model.cpp:764]: who will investigate this?

CC @jiwaszki

Comment (ilyachur) [reply src/bindings/python/src/pyopenvino/graph/model.cpp:764]: I created ticket for Python team

Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/model.cpp:764]: we have an issue connected with RTMap -- #98169

File: src/core/src/any.cpp (hunk 1 above)
Comment (ilyachur) [reply src/core/src/any.cpp:67]: Added test

**Pull request merged**